### PR TITLE
LPS-137810 buildLang

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by p2kmgcl** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/resources/META-INF/resources/applied_filters.jsp [hunk 1]
```diff
@@ -18,28 +18,51 @@
...
-		<span class="label-item label-item-after">
-			<button aria-label="<%= LanguageUtil.get(request, "remove-filter") %>" class="close remove-collection-applied-filter-button" data-filter-fragment-entry-link-id="<%= appliedFilter.get("filterFragmentEntryLinkId") %>" data-filter-type="<%= appliedFilter.get("filterType") %>" data-filter-value="<%= appliedFilter.get("filterValue") %>" type="button">
-				<span class="c-inner">
-					<clay:icon
-						symbol="times"
-					/>
+<div class="d-flex py-1" id="<%= collectionAppliedFiltersFragmentRendererDisplayContext.getFragmentEntryLinkNamespace() %>">
+	<div class="flex-grow-1">
+		<c:choose>
+			<c:when test="<%= collectionAppliedFiltersFragmentRendererDisplayContext.getAppliedFilters().isEmpty() %>">
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/resources/META-INF/resources/applied_filters.jsp:26]: I think you cannot chain here

Comment (p2kmgcl) [reply modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/resources/META-INF/resources/applied_filters.jsp:26]: But... what about happiness?

Comment (victorg1991) [reply modules/apps/fragment/fragment-renderer-collection-filter-impl/src/main/resources/META-INF/resources/applied_filters.jsp:26]: 404

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
See: https://github.com/liferay-echo/liferay-portal/pull/5597

**Pull request closed without merging**